mat/eigen.go

```diff
@@ -156,10 +156,15 @@ func (e *Eigen) Factorize(a Matrix, left, right bool) (ok bool) {
 	if left {
 		vl = *NewDense(r, r, nil)
 		jobvl = lapack.LeftEVCompute
+	} else {
+		vl.mat.Stride = 1
```

Comment:
I'm not happy about having to add these to `mat`, on the other hand I'd like to have the strides at least 1 as in the reference. Then `lapack64` seems like the right place to assure this, a connection between blas64 structs with zero default values and lapack with strides at least 1.

Reply:
This seems fair.